apps/gitness/src/components-v2/app-shell.tsx

```diff
@@ -226,19 +226,23 @@ export const AppShellMFE = () => {
 
   return (
     <>
-      <BreadcrumbsAndOutlet className="min-h-screen text-foreground-2" />
+      <BreadcrumbsAndOutlet className="text-foreground-2 min-h-screen" />
       <Toaster />
     </>
   )
 }
 
 function BreadcrumbsAndOutlet({ className }: { className?: string }) {
+  const { isInset } = useThemeStore()
+
   return (
-    <div className={cn('h-full flex flex-col', className)}>
-      <div className="layer-high sticky top-0 bg-background-1">
-        <Breadcrumbs />
+    <div className={cn('h-full', { 'overflow-hidden h-screen p-2 bg-sidebar-background-1': isInset })}>
+      <div className={cn('h-full flex flex-col', { 'rounded-md overflow-auto bg-background-1': isInset }, className)}>
```

Comment:
should the className from props be applied to the outermost div?
with this change, we will not be able to override the inset styles from outside (though is that even a requirement?)

Reply:
You're right here. Fixed👌